src/js/utils/create-deco.js

```diff
@@ -4,6 +4,7 @@ export default (pos, type) => {
   const span = document.createElement('span');
   span.classList.add('invisible', `invisible--${type}`);
   return Decoration.widget(pos, span, {
-    marks: []
+    marks: [],
+    key: type
```

Comment:
I'm not sure the param needs be named "type" here as ultimately it's being set to the `key`, so potentially it can just be called "key" to keep it nice and general if it needs to be used in other ways in future. Pedantic I know but am a stickler for readability!

Reply:
That's an old parameter. On line 5 it's used to create the class name.  Should I rename that to `key`?